src/main/java/chess/application/web/GameController.java

```diff
@@ -20,60 +26,91 @@ public GameController(GameService gameService) {
         this.gameService = gameService;
     }
 
-    public static void main(String[] args) {
-        SpringApplication.run(GameController.class, args);
-    }
-
     @GetMapping("/")
-    public String index(Model model) {
-        model.addAllAttributes(gameService.modelReady());
+    public String index() {
         return "index";
     }
 
-    @GetMapping("/start")
-    public String start(Model model) {
-        gameService.start();
-        return play(model);
+    @GetMapping("/games")
+    public ResponseEntity<String> list() {
+        String listData = jsonTransformer.render(gameService.list());
+        return ResponseEntity.ok().body(listData);
     }
 
-    @GetMapping("/load")
-    public String load(Model model) {
-        gameService.load();
-        return play(model);
+    @PostMapping("/game")
+    public ResponseEntity<String> create(@RequestParam String title, @RequestParam String password) {
+        try {
+            gameService.createRoom(title, password);
+        } catch (IllegalArgumentException e) {
+            return createMessageResponse(HttpStatus.BAD_REQUEST, e.getMessage());
+        }
+        return createMessageResponse(HttpStatus.CREATED, "방이 생성되었습니다.");
     }
 
-    @PostMapping("/move")
-    public String move(Model model, @RequestParam String source, @RequestParam String target) {
-        gameService.move(source, target);
-        if (gameService.isGameFinished()) {
-            return end(model);
+    @DeleteMapping("/game/{gameNo}")
+    public ResponseEntity<String> delete(@PathVariable long gameNo, @RequestBody String password) {
+        try {
+            gameService.checkPassword(gameNo, password);
+            gameService.delete(gameNo);
```

Comment:
컨트롤러에서 여러 메서드를 호출하기보다는 서비스의 메서드 하나에서 다 처리하도록 만들어주세요.

```suggestion
    try {
            gameService.delete(gameNo);
            return ResponseEntity.ok("방이 삭제되었습니다.");
    } catch (IllegalArgumentException e) {
```